specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -84,38 +84,47 @@ Power Management compliance.
   * It will subscribe to a In-memory Push event handler hanging off the retrieved Push Channel component.
     * The app should ideally handle the event by setting the Handled property to true to prevent Background Activation from <br/>
     launching a new process.
-
+* If the application is unpackaged, it will use the ProtocolActivator to register the activator instead. The app will also need to 
+call into the bootstrapper API to have access to the Windows App SDK. Click [here](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/tutorial-unpackaged-deployment?tabs=cpp) for information on how to build and 
+deploy unpackaged applications with Windows App SDK.
 ```cpp
 int main()
 {
-    // Register the COM Activator GUID
-    PushNotificationActivationInfo info(
-        PushNotificationRegistrationKind::PushTrigger | PushNotificationRegistrationKind::ComActivator,
+    PushNotificationActivationInfo info = nullptr;
+    if(PushNotificationManager::IsActivatorSupported(PushNotificationRegistrationOptions::PushTrigger | PushNotificationRegistrationOptions::ComActivator))
```

Comment:
Set this to a local bool and use it everywhere below.
For example,
bool isComActivationSupported = PushNotificationManager::IsActivatorSupported(PushNotificationRegistrationOptions::PushTrigger | PushNotificationRegistrationOptions::ComActivator);